project_name/tests/conftest.py

```diff
@@ -0,0 +1,3 @@
+print("Hello World")
+# print a smiley face
+print("\U0001f600")
```

Comment:
😄

Reply:
Give this a try - assuming your editor supports it:

```suggestion
print("😄")
```

Reply:
VSCode and pylance complains about this for me, but  maybe it works foryou

Reply:
No warnings on my end - and it runs in my terminal:
<img width="133" alt="image" src="https://user-images.githubusercontent.com/5139267/218578610-698d073d-c048-4fd4-a603-d6d0b78aa077.png">
What pylance error (warning?) is raised?

Reply:
Revisiting this the print statement was actually fine.  It was trying to assign a value to the emoji that complained, understandably.  (I forget why I expected this to work)
<img width="336" alt="image" src="https://user-images.githubusercontent.com/35379409/218808746-3e8d4be8-9a15-4ab9-8630-0c5607cee9a1.png">